ws-nextjs-app/pages/[service]/send/[id]/FormField/TextArea.tsx

```diff
@@ -15,29 +15,44 @@ export default ({
   label,
   hasAttemptedSubmit,
 }: InputProps) => {
-  const { isValid, value = '', required, wasInvalid } = inputState ?? {};
-  const translation = 'Maximum 500 Words';
+  const {
+    isValid,
+    value = '',
+    required,
+    wasInvalid,
+    wordLimit,
+  } = inputState ?? {};
+  const hasWordCount = !!wordLimit;
+  const translation = `Maximum ${wordLimit} Words`; // hardcoded
+  const describedByWordCount = `${id}-wordLimit`;
 
   return (
     <>
       <Label id={id}>{label}</Label>
-      <Paragraph
-        css={{ marginBottom: `${pixelsToRem(6)}rem` }}
-        fontVariant="sansRegular"
-        size="brevier"
-      >
-        {translation}
-      </Paragraph>
+      {hasWordCount && (
+        <Paragraph
+          css={{ marginBottom: `${pixelsToRem(6)}rem` }}
+          fontVariant="sansRegular"
+          size="brevier"
+          id={describedByWordCount}
+        >
+          {translation}
+        </Paragraph>
+      )}
       <textarea
         id={id}
         css={[styles.textField, styles.textArea, styles.focusIndicator]}
         name={name}
         value={value as string}
         onChange={e => handleChange(e.target.name, e.target.value)}
+        {...(hasWordCount && { 'aria-describedby': describedByWordCount })}
```

Comment:
Is it worth always announcing that there's a word limit, defaulting to english? Just thinking if someone is blind would it be announced in any way that they have run out of space while typing - not sure if having a default english would be more confusing for some people who dont speak it, just a thought !

Reply:
Heya - first up I realised I made this confusing using wordCount. I've updated this to WordLimit. 

I think to answer your question, no I don't think we always need to announce this. Since not all text areas have a word limit. I assume this is set by journalists since it comes through as an optional value in the data. (Is this a concern maybe?)

The original Morph implementation also had a 'word count' type functionality where it calulcates the words remaining as you type (e.g. 300/500 words remaining). I think this would be great to have for non-sighted users. But I guess isn't something we're doing in the MVP.